From 4eb9211e1a297f75c3f28067c893053ba51ba3db Mon Sep 17 00:00:00 2001
From: Alex Van Damme <atv@google.com>
Date: Mon, 24 Jun 2019 13:51:40 -0700
Subject: [PATCH 1/4] Unmark GstDebugColorFlags as flags

- This isn't compatible with older python3-gi, and regenerating the
introspection packages in a cross-compile environment is prohibitively
difficult.

Change-Id: I31bb2330cec6da0deba16e85550ea1e1d21a043c
---
 gst/gstinfo.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/gstinfo.h b/gst/gstinfo.h
index 82e4f9b23..e4aa13ab2 100644
--- a/gst/gstinfo.h
+++ b/gst/gstinfo.h
@@ -151,7 +151,7 @@ typedef enum {
  * These are some terminal style flags you can use when creating your
  * debugging categories to make them stand out in debugging output.
  */
-typedef enum { /*< flags >*/
+typedef enum {
   /* colors */
   GST_DEBUG_FG_BLACK		= 0x0000,
   GST_DEBUG_FG_RED		= 0x0001,
-- 
2.26.0

